docs/references/keywords.rst

```diff
@@ -76,6 +76,18 @@ Keywords
 ``license``
     A string specifying the license of the package.
 
+``license_file``
+
+    .. warning::
+        ``license_file`` is deprecated. Use ``license_files`` instead.
+
+``license_files``
+
+    A list of glob patterns for license related files that should be included.
+    If neither ``license_file`` nor ``license_files`` is specified, this option
+    defaults to ``LICEN[CS]E*``, ``COPYING*``, ``NOTICE*``, and ``AUTHORS*``.
+    Any ``exclude`` specified in ``MANIFEST.in`` will overwrite it.
```

Comment:
In my experience, `MANIFEST.in` only affects the sdist. Can you confirm that `exclude` in MANIFEST.in with `LICENSE` present has the same effect as not specifying a license on older Setuptools?

Reply:
I'm not exactly sure what you mean. AFAIK the change itself will only the `sdist` command. `bdist_wheel` will use the `license_file` and `license_files` fields or the default patterns. Therefore it's probably not wise to exclude a license file in the `MANIFEST.in` to begin with. But in case you only care about the source dist, any `exclude` does have president over a matched license file. There are two existing test cases that cover it.
I'm going to remove the last sentence, since it's probably more confusing than helpful especially to beginners.
https://github.com/pypa/setuptools/blob/743298103c05e17a22a74afdf5b2bf47630d2eb4/setuptools/tests/test_egg_info.py#L637-L655